content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,118 @@
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and is used to establish browser-to-server and browser-to-browser connections
+between applications. WebRTC serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying transport
+protocol.
+
+## WebRTC in libp2p
+
+### Key features
+
+- Peer connections: WebRTC enables [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes through a process called [signaling](https://webrtc.org/getting-started/peer-connections#signaling).
+  In non-decentralized WebRTC, this is facilitated by a [TURN server.](https://webrtc.org/getting-started/turn-server)
+  This allows for faster and more efficient communication.
+
+- Data channel: WebRTC also provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send text or binary data over an active connection to a peer.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/))to connect to nodes that run behind
+  NATs and firewalls.
+
+- [Security](../secure-comm/overview): WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
+
+Browsers expose an API to establish WebRTC connections. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate. WebRTC enables
+a node to send and receive streams through the `DataChannel`.
+
+### Browser-to-Server
+
+libp2p WebRTC enables browsers to connect to public server nodes (and eventually,
+browsers to connect to other browsers) without those endpoints providing a TLS
+certificate within the browser's trustchain.
+
+{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted CA (certificate authority). Few nodes have such a certificate, which is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC and WebTransport support encrypted communication without requiring a signed certificate from a trusted CA in the same way that WebSockets do." />}}
+
+When connecting to a WebSocket server, browsers require the server to present a
+TLS certificate signed by a trusted CA (certificate authority). Few nodes have such
+a certificate, so WebSocket never saw widespread adoption in the
+libp2p network.
```

Comment:
Isn't that already written in the previous paragraph?

Reply:
Yes